### DISCUSSION TITLE
Styling Enquiry

### DISCUSSION THREAD

**Discussion by willliamchan** [Clooos/Bubble-Card#526] [Questions about config, custom styles and templates] [answered]
I was surfing the issues section re styling and found we've got quite a few things like:-

.bubble-button-background
.bubble-bubble-card-container
.bubble-icon

Is there a way / place I can get the list of available options? I am kind of confuse the usage between .bubble-button-background and .bubble-bubble-card-container, looks like they both help changing color of bubble card background but what makes the differences? I found I need to use .bubble-button-background to tweak the color for a Fan entity and I need to use .bubble-bubble-card-container when it comes to Climate entity.

**Comment by MrBearPresident** [comment, marked as answer]
This you could find in the code and in the [readme](https://github.com/Clooos/Bubble-Card?tab=readme#bubble-card).
